pysteps/io/importers.py

```diff
@@ -135,6 +135,119 @@
     PYPROJ_IMPORTED = False
 
 
+def import_crri_eu(filename, **kwargs):
```

Comment:
I would prefer to keep an alphabetical order for the importers in this module.

Also, don't forget to add your method's name in the list of available importers in the docstrings at the beginning of the module (lines 75-82).